eng/pipelines/mgmt.yml

```diff
@@ -14,8 +14,8 @@ pr:
     - sdk/appplatform/Microsoft.Azure.Management.AppPlatform
     - sdk/attestation/Microsoft.Azure.Management.Attestation
     - sdk/authorization/Microsoft.Azure.Management.Authorization
-    - sdk/automation/Microsoft.Azure.Management.Automation
     - sdk/automanage/Microsoft.Azure.Management.Automanage
+    - sdk/automation/Microsoft.Azure.Management.Automation
     - sdk/avs/Microsoft.Azure.Management.Avs
```

Comment:
Please don't make other changes to this file.  Please just add your change (note that you might need to pull the latest from master if you did not make any changes).

Reply:
I just downloaded the yml from master.  copied over the version in my branch and added security insights.